### ISSUE TITLE
sounddevice Yamaha DME64N

### ISSUE THREAD

**Issue by MeandtheDevilknockingonyourdoor** [bitfocus/companion-module-requests#192]
I would like to control this device. It is important that I can switch scenes + recall and store functions. it is also important that the mute functions (mute all) can be used. I will be happy if it is successfull. Best Martin

**Comment by josephdadams** [comment]
Hello, you need to supply the link to the protocol information to control the device.

**Comment by MeandtheDevilknockingonyourdoor** [comment]
[Yamaha DME Remote Control Protocol](https://de.yamaha.com/files/download/other_assets/8/325448/dme_remote_ctrl_prtcl3102_en.pdf)

**Comment by MeestorX** [comment]
Do you have one of these that you can work currently? You would need to be willing to do some back and forth testing for a while.

**Comment by MeandtheDevilknockingonyourdoor** [comment]
Yes, in the future I can ask my company, maybe in two or three weeks.
Thx
Martin

Von meinem iPhone gesendet

**Comment by MeandtheDevilknockingonyourdoor** [comment]
Hey, i come back to my question from the last year, i have one to test it. 
Have you work on it?

**Comment by MeestorX** [comment]
Sorry, but I don't think I'll have time for a while.

**Comment by MeandtheDevilknockingonyourdoor** [comment]
Ok, thanks no problem. Hopefully in the future
